tools/resourcedocsgen/main.go

```diff
@@ -74,6 +75,10 @@ func main() {
 		mergeOverlaySchemaSpec(mainSpec, overlaySpec)
 	}
 
+	// Delete existing docs before generating new ones.
+	fmt.Printf("Deleting %s", outDir)
+	os.RemoveAll(outDir)
```

Comment:
Should we check that this doesn't return an error before continuing?

Reply:
Ah, I didn't even notice that, thanks, yeah. Will add.